news/2019-09-07-community-mentorship-program-fall-2020-mentee-signups-now-open.md

```diff
@@ -0,0 +1,37 @@
+---
+layout: post
+title: "Community Mentorship Program Fall 2020 Signups Now Open"
+date: 2019-09-08 12:50:00 +0000
+---
+
+The next cycle of the Community Mentorship program is starting again! If you've ever found yourself wanting to get into beatmapping, or expanding your current knowledge and skills, this program will serve as a great opportunity for you!
+
+![](/wiki/shared/news/banners/community-mentorship-program.jpg)
+
+The Community Mentorship program serves as place for upcoming mappers and modders to develop their skills underneath the guidance of more experienced mappers and modders in the community. Over the course of a cycle, you'll have the opportunity to learn from your mentor, participate in various mentorship events, such as mapping contests, hang out with fellow mentees and more!
+
+For those of you already experienced in mapping, and have a desire to take those that are newer under your wing, mentor applications are already open! In the forum announcement and rule thread [here](https://osu.ppy.sh/community/forums/topics/955356), you'll be able to apply if you’re considering mentoring! Keep in mind mentor applications take sometime to process, so please be patient.
+
+For this upcoming fall cycle, we are looking for both mentors and mentees for all game modes!
+
+Since last cycle, we've restructured our general mentorship timeline.
+
+- No longer are mentors required to sign-up each cycle, instead a mentor may mentor for as long as they'd like, but can retire at any time, once completing their commitments to their mentees.
+
+- Mentee applications are accepted every 4-weeks, and expire after 8, if they have not been accepted. However, applicants with high interests are encouraged to reapply as often as possible, to better reflect their interest, changes in mentor preferences, and availability.
+
+- Mentors and mentees are fixed for a minimum 8 weeks, but mentors can also hold mentees for longer than this, and are encouraged to do so.
+
+For this following cycle, the timeline is as follows (all opening and closing dates start and end at midnight UTC+0):
```

Comment:
I think adding one before this paragraph would be nice as well.
```
- Mentors and mentees are fixed for a minimum 8 weeks, but mentors can also hold mentees for longer than this, and are encouraged to do so.

---

For this following cycle, the timeline is as follows (all opening and closing dates start and end at midnight UTC+0):
```

Reply:
sure